tfexec/errors.go

```diff
@@ -26,7 +26,9 @@ var (
 )
 
 func parseError(err error, stderr string) error {
-	if _, ok := err.(*exec.ExitError); !ok {
+	// if not an ExitError, just return it, do not try to parse
+	var ee *exec.ExitError
+	if !errors.As(err, &ee) {
```

Comment:
we should probably do this manually to support 0.12 compilation (if not testing)